src/coreclr/tools/aot/ILCompiler/Program.cs

```diff
@@ -688,7 +687,9 @@ private static IEnumerable<int> ProcessWarningCodes(IEnumerable<string> warningC
         {
             foreach (string value in warningCodes)
             {
-                string[] values = value.Split(s_separators, StringSplitOptions.RemoveEmptyEntries);
+#pragma warning disable CA1861 // Avoid constant arrays as arguments. Only executed once.
```

Comment:
Is this actually executed once? It's inside the body of a foreach.

Reply:
Good catch. It should be outside the loop.

Reply:
Or it can be a static field.